packages/orbit-components/src/Wizard/index.js

```diff
@@ -0,0 +1,154 @@
+    }
+    return status;
+  });
+  // $FlowFixMe: not sure why "props" evaluates to "mixed"
+  const activeStepTitle = childrenArray.find((step, index) => index === activeStep)?.props.title;
+  const stepsCount = React.Children.count(children);
+
+  const steps = React.Children.map(children, (step, index) => (
+    <WizardStepContext.Provider
+      value={{
+        index,
+        status: stepStatuses[index],
+        nextStepStatus: stepStatuses[index + 1],
+        isActive: activeStep === index,
+        onChangeStep,
+      }}
+    >
+      {step}
+    </WizardStepContext.Provider>
+  ));
+
+  if (!isTablet) {
+    return (
+      <>
+        <div
+          css={css`
+            button {
+              /* it's intended to go from edge to edge */
+              border-radius: 0;
+            }
+          `}
+        >
+          <ButtonLink
+            ref={toggle}
+            dataTest={dataTest}
+            ariaControls={id}
+            ariaExpanded={open}
+            type="secondary"
+            fullWidth
+            iconRight={<ChevronDown />}
+            onClick={() => {
+              setOpen(true);
+            }}
+          >
+            <Stack as="span" inline>
+              <b>{`${activeStep + 1} of ${stepsCount}`}</b>{" "}
+              <span
+                css={css`
+                  font-weight: normal;
+                `}
+              >
+                {activeStepTitle}
+              </span>
+            </Stack>
+          </ButtonLink>
+        </div>
+        <Portal>
+          <div id={id}>
+            {open && (
+              <Modal>
+                <nav
+                  css={css`
+                    /* matching this to ModalBody's border-radius */
+                    padding-top: 9px;
+                  `}
+                >
+                  <ul
+                    css={css`
+                      list-style-type: none;
+                      margin: 0;
+                      padding: 0;
+                      > * + * {
+                        border-top: 1px solid ${theme.orbit.paletteCloudDark};
+                      }
+                    `}
+                  >
+                    {steps}
+                    <li>
+                      <CardSection>
+                        <Button
+                          type="secondary"
+                          fullWidth
+                          onClick={() => {
+                            setOpen(false);
+                          }}
+                        >
+                          Close
+                        </Button>
+                      </CardSection>
+                    </li>
+                  </ul>
+                </nav>
+              </Modal>
+            )}
+          </div>
+        </Portal>
+      </>
+    );
+  }
+
+  return (
+    <nav>
+      <ul
+        css={css`
+          list-style-type: none;
+          padding: 0;
+          margin: 0;
+          display: flex;
+          > * {
+            flex: 1 1 0%;
```

Comment:
why? you know, that all childs are `li`, select them by type, not with `>` selector

Reply:
I'm fine with either, there is a slight difference in specificity, but it's not important for this use case.

Reply:
sure, it will not have any significant value on perfomance, just good tone to use more performant css selectors